lib/nexmo_developer/app/presenters/page_title.rb

```diff
@@ -1,19 +1,59 @@
 class PageTitle
+  WORD_FILE = "#{Rails.configuration.docs_base_path}/config/normalized_words.yml".freeze
+
   def initialize(product, document_title)
     @product        = product
     @document_title = document_title
+
+    after_initialize!
+  end
+
+  def after_initialize!
+    raise 'You must provide a config/normalized_words.yml file in your documentation path.' unless File.exist?(WORD_FILE)
```

Comment:
Can we make this file optional? Not all business units will have normalised words

Reply:
Yes, will do.